driver-scala/src/main/scala/org/mongodb/scala/MongoCredential.scala

```diff
@@ -169,7 +169,7 @@ object MongoCredential {
    *
    * @param userName the user name
    * @return the credential
-   * @see [[http://docs.mongodb.org/manual/core/authentication/#x-509-certificate-authentication X-509]]
+   * @see [[https://www.mongodb.com/docs/manual/core/authentication/#x-509-certificate-authentication X-509]]
```

Comment:
Looks like this links to an old target.
New link suggestion: https://www.mongodb.com/docs/manual/core/security-x.509/